internal/sync/manager.go

```
@@ -0,0 +1,209 @@
+}
+
+// processBufferedNotifications sends any notifications that were queued before initialization
+func (m *Manager) processBufferedNotifications() {
+	m.bufferMutex.Lock()
+	buffer := m.notifBuffer
+	m.notifBuffer = nil
+	m.bufferMutex.Unlock()
+
+	for _, notification := range buffer {
+		if m.isPrimary && m.server != nil {
+			m.server.Broadcast(notification)
```

golangci-lint (1.64.8)

Error return value of `m.server.Broadcast` is not checked

(errcheck)